### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by ShengChih-Lin**
Student Number:45427804
Student Name: ShengChih Lin
Course: Comp3710

Task:  Segment the OASIS brain data set with an Improved UNet with all labels having a minimum Dice similarity coefficient of 0.9 on the test set.

Please read my Readme.md which contains my summary.

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be a LOT more specific
- Code valid for Easy Difficulty
- Good results

Other Feedback
- gitignore all DS_Store files
- i would recommend that you tidy up the main file in the future

**Comment by rexsw** [comment]
Code looks good for easy difficulty

For next time;

Try to only create fewer more meaningful commits  

Tensorflow has functions for image opening and more so there’s no need to also use PIL in your project. 

!unzip '/gdrive/My Drive/keras_png_slices_data.zip' -d '/gdrive/My Drive/comp3710project' is not vaild code in a .py file

Paths such as '/gdrive/My Drive/comp3710project/keras_png_slices_data/keras_png_slices_seg_train/*.png' won’t work for the user and should either be set dynamical or it needs to be made clear to the user that they need to change these paths

Code in files like driver scripts should be in a main function and in a python file.

The code for creating the UNET should be in a function or class

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Revert the change to the main Readme file of the course project.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.
